audpsychometric/__init__.py

```diff
@@ -2,11 +2,11 @@
 from audpsychometric.core import datasets
 from audpsychometric.core.datasets import list_datasets
 from audpsychometric.core.datasets import read_dataset
+from audpsychometric.core.gold_standard import agreement_categorical
+from audpsychometric.core.gold_standard import agreement_numerical
 from audpsychometric.core.gold_standard import evaluator_weighted_estimator
-from audpsychometric.core.gold_standard import gold_standard_mean
-from audpsychometric.core.gold_standard import gold_standard_median
-from audpsychometric.core.gold_standard import gold_standard_mode
-from audpsychometric.core.gold_standard import rater_confidence_pearson
+from audpsychometric.core.gold_standard import mode
+from audpsychometric.core.gold_standard import rater_agreement_pearson
```

Comment:
It is not a new function, it was part of `gold_standard.py` before.
The difference to the other functions in `reliability.py` is that it does not return a single value, but one value per rater. Honestly, I don't know where to put it ;)

Reply:
One solution is to remove the grouping we do for the API documentation:

![image](https://github.com/user-attachments/assets/d8b9e878-49b7-4326-ada3-e258b1a80418)

Then we don't need to agree where to put it and can just list everything alphabetically.

Reply:
I think, it will be easier to discuss these in another pull request. I will merge here now and continue. Feel free to open issues for remaining questions.

Reply:
I think this has been opened already. It can be resolved here.